src/main/groovy/org/jetbrains/intellij/Utils.groovy

```diff
@@ -85,19 +85,31 @@ class Utils {
         result
     }
 
+    static def configDir(@NotNull String sandboxDirectoryPath, String suffix) {
```

Comment:
I think all these methods might be inlined now

Reply:
they are invoked from two places. But I deleted old methods which is not using now

Reply:
But this is just concatenation, here is no logic anymore

Reply:
fixed